Client/Frontend/Settings/AppSettingsTableViewController.swift

```diff
@@ -121,6 +121,7 @@ class AppSettingsTableViewController: SettingsTableViewController, FeatureFlagga
             generalSettings.insert(SearchBarSetting(settings: self), at: 5)
         }
 
+        let autofillCreditCardStatus = featureFlags.isFeatureEnabled(.creditCardAutofillStatus, checking: .buildOnly)
```

Comment:
@adudenamedruby could you confirm if the nimbus settings are as follows then the value of `autofillCreditCardStatus` is true?

I am getting false here so want to double-check the featureflag setup

<img width="1071" alt="image" src="https://user-images.githubusercontent.com/8919439/209736014-b5347899-c756-47e3-8849-b500937edbaa.png">

Reply:
These settings will return `false` for prod and `true` for dev & beta.